tools/ci_build/github/azure-pipelines/linux-gpu-ci-pipeline.yml

```diff
@@ -87,6 +87,51 @@ jobs:
       artifactName: 'drop-linux'
       targetPath: '$(Build.BinariesDirectory)/Release'
 
+  - task: CmdLine@2
+    inputs:
+      script: |
+         set -e -x
+         #We assume the machine doesn't gcc and python development header files
+         sudo rm -f /build /onnxruntime_src
+         sudo ln -s $(Build.SourcesDirectory) /onnxruntime_src
+         python3 -m pip uninstall -y ort-gpu-nightly ort_nightly onnxruntime onnxruntime-gpu -qq
+         cp $(Build.SourcesDirectory)/tools/ci_build/github/linux/docker/scripts/manylinux/requirements.txt $(Build.BinariesDirectory)/requirements.txt
+         #Test ORT with the latest ONNX release.
+         sed -i 's/git+http:\/\/github\.com\/onnx\/onnx.*/onnx==1.9.0/' $(Build.BinariesDirectory)/requirements.txt
+         python3 -m pip install -r $(Build.BinariesDirectory)/requirements.txt
+         python3 -m pip install $(Build.BinariesDirectory)/Release/dist/*.whl
+         ln -s /data/models $(Build.BinariesDirectory)
+         cd $(Build.BinariesDirectory)/Release
+         #Restore file permissions
+         xargs -a $(Build.BinariesDirectory)/Release/perms.txt chmod a+x
+         cd $(Build.SourcesDirectory)/java
+         /usr/local/gradle/bin/gradle "cmakeCheck" "-DcmakeBuildDir=$(Build.BinariesDirectory)/Release" "-DUSE_CUDA=1"
+         cd /tmp
+         python3 $(Build.SourcesDirectory)/tools/ci_build/build.py \
+              --build_dir $(Build.BinariesDirectory) --cmake_generator Ninja \
+              --config Release --test \
+              --skip_submodule_sync \
+              --build_shared_lib \
+              --parallel \
+              --build_wheel \
+              --enable_onnx_tests --use_cuda --cuda_version=11.1 --cuda_home=/usr/local/cuda-11.1 --cudnn_home=/usr/local/cuda-11.1 \
+              --enable_pybind --build_java --ctest_path ''
+
+              - job: Linux_Test
```

Comment:
Indentation matters.  

You may try to manually queue a build for this pipeline, I'm afraid this yaml change doesn't work.